testplan/common/utils/match.py

```diff
@@ -155,7 +237,7 @@ def match(
                 else:
                     break
 
-            self.position = log.tell()
+            self.position = self.log_stream.position
```

Comment:
Also do we need to maintain the current position at LogMatch level now that the log_stream object also keeps record of current position. Maybe it's good enough to just keep a few marks at LogMatcher level? and delegate to log_stream to start from current position.

Reply:
log matcher keeps the position for continuous read, however if the stream is closed and reopened it is not maintained, taht I want to keep for now, see above.